sign_language_datasets/datasets/dgs_corpus/dgs_corpus.py

```diff
@@ -2,12 +2,17 @@
 
 import gzip
 import json
-from os import path
 
 import numpy as np
 import tensorflow as tf
 import tensorflow_datasets as tfds
-from pose_format.utils.openpose import load_openpose
+
+from os import path
+from typing import Dict, Any, Set, Optional
+# noinspection PyUnresolvedReferences
+from pose_format.utils.openpose import load_openpose, OpenPoseFrame, OpenPoseFrames
+# noinspection PyUnresolvedReferences
```

Comment:
why are these `noinspection PyUnresolvedReferences` needed? The package should be installed and have these references

Reply:
that's correct, I should have removed them. I added a commit